app/assets/stylesheets/styles/pages.scss

```diff
@@ -244,25 +258,25 @@
 
       .material-icons {
         background-color: #42a49a;
+        color: #fff;
         padding: 10px;
         position: relative;
+        transition-duration: 0.6s;
 
-        &::before {
-          content: "";
-          position: absolute;
-          width: 100%;
-          height: 100%;
-          background-color: #fff;
-          transform: rotate(8deg);
+        &:hover {
```

Comment:
Nesting should be no greater than 4, but was 5<br>Selector should have depth of applicability no greater than 2, but was 4